cmd/validate-yaml/main.go

```diff
@@ -97,3 +119,50 @@ func PrintValidationResults(printer IPrinter, invalidFiles []*pkgExtractor.Inval
 	validFilesCount := filesCount - len(invalidFiles)
 	printer.PrintYamlValidationSummary(validFilesCount, filesCount)
 }
+
+func SendResults(localConfig ILocalConfig, sender ICliClient, cliVersion string, isValid bool, invalidYamlFiles []*pkgExtractor.InvalidFile, filesPaths []string) {
+	osInfo := utils.NewOSInfo()
+	resultFiles := prepareResultFiles(invalidYamlFiles, filesPaths)
+	configData, err := localConfig.GetLocalConfiguration()
+	if err != nil {
+		return
+	}
+	var status string = STATUS_PASSED
+	if !isValid {
+		status = STATUS_FAILED
+	}
+	result := &cliClient.ValidatedYamlResult{
+		Token:    configData.Token,
+		ClientId: configData.ClientId,
+		Files:    resultFiles,
+		Status:   status,
+		Metadata: &cliClient.Metadata{
+			CliVersion:      cliVersion,
+			Os:              osInfo.OS,
+			KernelVersion:   osInfo.KernelVersion,
+			PlatformVersion: osInfo.PlatformVersion,
+		},
+	}
+
+	sender.SendValidateYamlResult(result)
```

Comment:
I cannot use the name `cliClient` because I'm overriding the package's name in the same function (`&cliClient.ValidatedYamlResult`), changed the name to `client`, what do you think?